eZ/Publish/Core/settings/fieldtype_services.yml

```diff
@@ -54,3 +54,4 @@ services:
             $locationService: '@ezpublish.api.service.location'
             $contentTypeService: '@ezpublish.api.service.content_type'
             $mappings: '$fieldtypes.ezimageasset.mappings$'
+
```

Comment:
Nitpick: seems unnecessary.

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2632/commits/253648a428361200d25718ae008f56d07b1abdc8